mypage/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: layouts/base.njk
+title: My Page
+---
+
+# How to add a page to the site
+
+### 1. Create a [https://www.markdowntutorial.com/](Markdown) file with an ".md" file extension.
+
+The markdown file can be in the root folder of the site, an existing folder such as "/about", or in a new folder. A new folder is useful if you have multiple related pages--`index.md` will be the default page for the folder. The URL for the page will reflect where you place the file. For example, if a "/mypage.md" file is added to the root folder, then it will be available at:
+[http://localhost:8080/Tech-Writing-Project/mypage/](http://localhost:8080/Tech-Writing-Project/mypage/)
+
+    
+### 2. Add meta data to the top of the markdown file
+
+Metadata for the page, such as the page's `title` which appears in the browser tab, is placed at the top of the file denoted with starting and ending `---` markers. 
+
+Add the following properties to the metatdata:
+
+|Property| Example value | Purpose |
+|---|---|---|
+| title: | My Page! | Title to appear in the browser tab |
+| layout: | layouts/base.njk | Any of the layout files in `_includes/layouts` such as `base`, `home`, or `post` |  
+| eleventyNavigation:  | key: Fetch  order: 4 | Optional metadata for adding page to site's top navigagion. `key` provides the link text, and `order` speccifies the placement in navigation |
+| tags: | post | Used to associate pages into a collection. See Markdwn files in `posts/` as an example |
+
+All that is required is a title and layout:
+
+```yaml
+--- 
+layout: layouts/base.njk
+title: My Page
+---
+```
+
+### 3. Add page content
+
+The remainder of the file is Markdown that will be transformed into `html` for browsing. Add page content to the file.  
+
+
+```markdown
+--- 
+layout: layouts/base.njk
+title: My Page
+---
+
+# Cats vs. Dogs
+
+Cats!
+
+## What about adding html?
+
+<p>Sure, if you need it, but markdown is the preferred format.</p>
```

Comment:
So, this means I shouldn't use html unless there's some special reason that Markdown cant help me with?

Reply:
markdown -> Markdown